backend.native/tests/runtime/workers/worker10.kt

```diff
@@ -0,0 +1,57 @@
+package runtime.workers.worker10
+
+import kotlin.test.*
+
+import kotlin.native.concurrent.*
+
+class Data(val x: Int)
+
+val topInt = 1
+val topString = "string"
```

Comment:
It also makes sense to test for `var` `String`.

Reply:
Done.